migrations/deployed.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
This file is meant to be committed to git. It stores the deployed addresses whenever we do a deploy to kovan

Reply:
And will later be used for mainnet as well